test/services/PlaybookServiceTest.js

```diff
@@ -3,239 +3,265 @@ import { expect } from 'chai'
 import moment from 'moment'
 import * as playbook from '../../src/services/playbookService'
 
-describe('TestCondition', function () {
```

Comment:
This large diff is because I wrapped all the existing describe statements in a single describe statement to make it more obvious that test failures are related to PlaybookServiceTest.